srcpkgs/pinebookpro-kernel/template

```diff
@@ -0,0 +1,253 @@
+# Template file for 'pinebookpro-kernel'
+pkgname=pinebookpro-kernel
+version=5.5.0
+revision=1
+_commit=799b9141e48783a0844187ad00855b3d53f77998
+wrksrc="linux-pinebook-pro-${_commit}"
+short_desc="Linux kernel for Pinebook Pro (${version%.*} series)"
+maintainer="Renato Aguiar <renato@renatoaguiar.net>"
+license="GPL-2.0-only"
+homepage="https://www.kernel.org"
+distfiles="https://gitlab.manjaro.org/tsys/linux-pinebook-pro/-/archive/${_commit}/linux-pinebook-pro-${_commit}.tar.gz"
```

Comment:
I would prefer to not depend on any other distribution... but ok for now.

Reply:
oh and btw you can remove the ppc patches, this linux kernel tree is specific for aarch64, so that we could remove !aarch64 stuff...

Reply:
Agreed.

It seems they are trying to upstream their changes. I'll keep an eye on it and remove this dependency as soon possible.

Reply:
> oh and btw you can remove the ppc patches, this linux kernel tree is specific for aarch64, so that we could remove !aarch64 stuff...

That makes sense. I just pushed an updated patch.